modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/src/main/resources/META-INF/resources/js/components/card/CardList.es.js

```diff
@@ -80,7 +79,26 @@ const chartFactory = ({
 				/>
 			);
 		}
+		case 'checkbox': {
+			const newValues = Object.keys(values).reduce(
+				(acc, key) => ({
+					...acc,
+					...{
+						[key == 'true'
+							? Liferay.Language.get('true')
+							: Liferay.Language.get('false')]: values[key],
+					},
+				}),
+				{}
+			);
```

Comment:
Hey Milton, I'll make the changes. But I think it needs some adjustments for it to work:

`const newValues = {};
for (const [key, value] of Object.entries(values)) {
	const newKey = key === 'true' ? Liferay.Language.get('true') : Liferay.Language.get('false');
	newValues[newKey] = value;
}`

I replaced `newValues[keyValue]` with `newValues[newKey]`, since I think that was your intended chage.

Also, `Liferay.Language.get()` requires a constant value, so it can't take an expression as a value.

Reply:
```suggestion
	const newValues = {};
	for (const [key, value] of Object.entries(values)) {
		const newKey = key === 'true' ? Liferay.Language.get('true') : Liferay.Language.get('false');
		newValues[newKey] = value;
	}
```

Hey Milton, I'll make the changes. But I think it needs some adjustments for it to work:

I replaced `newValues[keyValue]` with `newValues[newKey]`, since I think that was your intended change.

Also, `Liferay.Language.get()` requires a constant value, so it can't take an expression as a value.